src/libcrun/cgroup.c

```diff
@@ -2186,41 +2164,10 @@ libcrun_cgroup_killall (const char *path, libcrun_error_t *err)
 }
 
 int
-libcrun_cgroup_destroy (const char *id, const char *path, const char *scope, int manager, libcrun_error_t *err)
+libcrun_cgroup_destroy_subsystem (const char *path, int mode, libcrun_error_t *err)
```

Comment:
the function is destroying more than just a subsystem.

Can we rename it to something like `do_cgroup_destroy` and make it static?